lib/Parser/DocumentParser.php

```diff
@@ -479,11 +508,17 @@ private function flush(): void
                     /** @var string[] $lines */
                     $lines = $this->buffer->getLines();
 
-                    $blockNode = $this->nodeFactory->createBlockNode($lines);
+                    $node = $this->nodeFactory->createBlockNode($lines);
 
-                    $document = $this->parser->getSubParser()->parseLocal($blockNode->getValue());
+                    // This means we are in an indented area that is not a code block.
```

Comment:
```suggestion
                    // This means we are in an indented area that is not a code block or definition list
```

Definitions lists are discovered by an indented line directly following a line with text. That is the only way it's distinguished from block quotes (which much be preceded by an empty line).

Reply:
I can see our knowledge about the library starting to intersect and help each other improve. Wonderful!